Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/A11yTabIndex.xaml

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="Xamarin.Forms.Controls.Issues.A11yTabIndex">
+	<Grid RowSpacing="2">
+
+		<Grid.ColumnDefinitions>
+			<ColumnDefinition></ColumnDefinition>
+		</Grid.ColumnDefinitions>
+
+		<Grid.RowDefinitions>
+			<RowDefinition Height="50"></RowDefinition>
+			<RowDefinition Height="50"></RowDefinition>
+			<RowDefinition Height="50"></RowDefinition>
+			<RowDefinition Height="Auto"></RowDefinition>
+			<RowDefinition Height="Auto"></RowDefinition>
+		</Grid.RowDefinitions>
+
+		<BoxView Color="Blue"></BoxView>
+		<Label Text="This text should be read when the page loads." TabIndex="-2" TextColor="White" />
```

Comment:
Update instructions to match how this runs in our control gallery.

From my testing in the control gallery when navigating to this page nothing is read to the user. Is that correct? 
First swipe says "Main Page main page help text" 
Second swipe reads back button
third swipe reads this text

Testing on Android S9

Reply:
The behavior swiping back and forth is a bit interesting on Android
- onload nothing plays
- swipe right "Main Page"
- swipe right "Gallery Button"
- swipe left "Main Page" 
- swipe left - *some sort of thunk sound?*
- swipe right "Gallery Button"

Not sure if it's related to this PR or not but figured I'd mention it

Reply:
Update instructions to match how this runs in our control gallery.

From my testing in the control gallery when navigating to this page nothing is read to the user. Is that correct? 
First swipe says "Main Page main page help text" 
Second swipe reads back button
third swipe reads this text

Testing on Android S9


The behavior swiping back and forth is a bit interesting on Android
- onload nothing plays
- swipe right "Main Page"
- swipe right "Gallery Button"
- swipe left "Main Page" 
- swipe left - *some sort of thunk sound?*
- swipe right "Gallery Button"

Not sure if it's related to this PR or not but figured I'd mention it